.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,39 @@
+name: CIFuzz
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+permissions: {}
+jobs:
+  Fuzzing:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+    - name: Build Fuzzers
+      id: build
+      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+      with:
+        oss-fuzz-project-name: 'jupytext'
+        language: python
+    - name: Run Fuzzers
+      uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
+      with:
+        oss-fuzz-project-name: 'jupytext'
+        language: python
+        fuzz-seconds: 800
```

Comment:
Is that the expected duration of the task on the CI? (if so that's fine, just curious about the duration)

Reply:
We have some wiggle room with this value, this is just what I have typically used - the default is 600.

From the docs:

"fuzz-seconds: Determines how long CIFuzz spends fuzzing your project in seconds. The default is 600 seconds. The GitHub Actions max run time is 21600 seconds (6 hours). This variable is only meaningful when supplied to the run_fuzzers action, not the build_fuzzers action."

Do you want to adjust it? Keep in mind, this will block the PR until it completes.